frontend/src/app/analytics/Analytics.tsx

```diff
@@ -8,23 +8,44 @@ import { LoadingCard } from "../commonComponents/LoadingCard/LoadingCard";
 
 import "./Analytics.scss";
 
-export const getDateFromDaysAgo = (daysAgo: number): Date => {
+const getDateFromDaysAgo = (daysAgo: number): Date => {
   const date = new Date();
   date.setDate(date.getDate() - daysAgo);
   return date;
 };
 
-export const getDateStringFromDaysAgo = (daysAgo: number): string => {
-  return getDateFromDaysAgo(daysAgo).toLocaleDateString();
+export const setStartTimeForDateRange = (date: Date): Date => {
+  date.setHours(0);
+  date.setMinutes(0);
+  date.setSeconds(0);
+  return date;
+};
+
+export const setEndTimeForDateRange = (date: Date): Date => {
+  date.setHours(23);
+  date.setMinutes(59);
+  date.setSeconds(59);
```

Comment:
Do you know how this interacts for end dates of "today"? I'm wondering if the backend will have any issues processing a slightly future date, i.e. if you're searching at 2pm and the end time is set to midnight.

Reply:
I wondered this myself! I'm not seeing any issues locally, and SQL specifically allows you to do this, so I think this is ok. I was trying to avoid introducing new bugs by using an `if`/`else` for local time.

Reply:
Sounds good! I was going to suggest adding a backend test for it, but it looks like all the backend tests already add 3 minutes to the end time anyways: https://github.com/CDCgov/prime-simplereport/blob/1310a4141b3dfc9707a6b9843b91467cde2e8e42/backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java#L1228